horqrux/circuit.py

```diff
@@ -58,11 +62,56 @@ def hea(n_qubits: int, n_layers: int, rot_fns: list[Callable] = [RX, RY, RX]) ->
     for _ in range(n_layers):
         for i in range(n_qubits):
             ops = [
-                fn(str(uuid4()), qubit)
+                fn("v_" + str(uuid4()), qubit)
```

Comment:
Why doing this ?

Reply:
no clue yet 😄

Reply:
@RolandMacDoland eventually, it's a convention for variational vs non variational parameters. I think I will change this a bit.

Reply:
@RolandMacDoland changed this by adding an extra field to specify feature map parameters from variational ones.